ckeditor/ckeditor5-dev
# [JSDoc] [WIP] Fixed inheritance system, filtered out internal doclets

### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Fixed inheritance mechanism and filtered out internal doclets. Closes #465.

---

### Additional information

* [x] Tests covering that changes (all old tests are passing now)
* [x] Improve API docs and add some internal comments

How to test:
1. Go to `ckeditor5`, ensure that all is up to date and run `yarn run switch-to-dev-dev`.
2. In `ckeditor5-dev` check out the `t/465`.
3. In `ckeditor5` run `yarn docs --skip-snippets [--skip-validation]`.


I've added TS checked API docs, it helped me much to understand which variable is which type.
